lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -59,19 +59,19 @@ module SharedCourseConstants
   ).freeze
 
   # All the categories options used to group course offerings in the assignment dropdown
-  COURSE_OFFERING_CATEGORIES = OpenStruct.new(
-    pl_self_paced: 'Self-Paced Professional Learning',
-    pl_virtual: 'Virtual Professional Learning',
-    pl_other: 'Other Professional Learning',
-    full_course: 'Full Courses',
-    csf: 'CS Fundamentals',
-    csc: 'CS Connections',
-    aiml: 'AI/ML',
-    hoc: 'Hour of Code',
-    csf_international: 'CS Fundamentals International',
-    math: 'Math',
-    twenty_hour: '20-hour',
-    other: 'Other'
+  COURSE_OFFERING_CATEGORIES = %w(
```

Comment:
since time is short, I'll just say that I think this is a good situation to ask this question, but I don't see a timing problem with the launch. the problem scenario is when old client code makes an ajax request and hits new server code which gives an unexpected response, which I don't see happening here.

Reply:
Awesome thank you!